### PR TITLE
Workflows - main screen - content area

### PR DESCRIPTION
**Pull request by jozefbakus**
implements: #97

### CODE REVIEW

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    <div class="btn-group">
+      <button type="button" class="btn btn-sm btn-secondary"><clr-icon shape="plus"></clr-icon> Create new workflow </button>
+    </div>
+  </clr-dg-action-bar>
+
+  <clr-dg-column>Workflow name</clr-dg-column>
+  <clr-dg-column>Project name</clr-dg-column>
+  <clr-dg-column>Is active</clr-dg-column>
+
+  <clr-dg-row *clrDgItems="let workflow of workflows" [clrDgItem]="workflow">
```
Comment (kevinwallimann) [inline ui/src/app/components/workflows/workflows-home/workflows-home.component.html:13]: Aren't these actions usually on the right side? I've seen three dots on the left side mostly to drag a row for reordering.

File: ui/src/app/stores/workflows/workflows.effects.spec.ts [hunk 2]
```diff
@@ -75,7 +75,7 @@ describe('WorkflowsEffects', () => {
       const getProjectsResponse = cold('-a|', { a: projects });
       const expected = cold('--a', { a: {
           type: WorkflowsActions.INITIALIZE_WORKFLOWS_SUCCESS,
-          payload: {projects: projects, workflows: [].concat(projects.map((project) => project.workflows))}
+          payload: {projects: projects, workflows: [].concat(...projects.map((project) => project.workflows))}
```
Comment (kevinwallimann) [inline ui/src/app/stores/workflows/workflows.effects.spec.ts:78]: Why do you need to `...` here?

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html (hunk 1 above)
Comment (jozefbakus) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html:13]: Usually yes. In clarity no. No matter where I place actions they are rendered on the left side.

File: ui/src/app/stores/workflows/workflows.effects.spec.ts (hunk 2 above)
Comment (jozefbakus) [reply ui/src/app/stores/workflows/workflows.effects.spec.ts:78]: It splits array of arrays into this concat(...items: ConcatArray<T>[])

File: ui/src/app/components/workflows/workflows-home/workflows-home.component.html (hunk 1 above)
Comment (kevinwallimann) [reply ui/src/app/components/workflows/workflows-home/workflows-home.component.html:13]: Hmm yes maybe they want to be consistent with the checkboxes (in batch action) which are usually on the left

**Review by kevinwallimann — APPROVED** [review]

**Pull request merged**